### PR TITLE
Updated lock.py for battery updates

### PR DESCRIPTION
**Pull request by ninadpchaudhari**
WRT : https://github.com/JoshuaMulliken/ha-wyzeapi/issues/333#issuecomment-1094341909 

Changing lock battery to lock_battery & keypad Battery to keypad_battery

### CODE REVIEW

**Comment by ninadpchaudhari** [comment]
tagging @yoinx  for confirmation

**Comment by yoinx** [comment]
The part that had made it into the codebase in the comment thread you linked was simply removing the "%" from the value. I still disagree with changing the name to include an underscore. The are already entities for the batteries for people to get the values from. This dev_attr list is simply for display purposes as far as I can tell. Ain't the underscore makes it display with an underscore in the device attributes display.

Bottom line... People should be polling the proper entities for these values, not this attributes field. These battery percentages shouldn't even actually exist here as I explained in the linked conversation.

**Comment by ninadpchaudhari** [comment]
Dear @JoshuaMulliken   I share my opinion with @yoinx 
But after some further reading, seems this behavior is Specifically mentioned to be avoided in the developer documentation! 

>Entities that generate a significant amount of state changes can quickly increase the size of the database when the extra_state_attributes also change frequently. Minimize the number of extra_state_attributes for these entities by removing non-critical attributes or creating additional sensor entities.

Source: [developer documentation about extra attributes! ](https://developers.home-assistant.io/docs/core/entity/#:~:text=changeable.%20Learn%20more.-,WARNING,-Entities%20that%20generate )
Check out the "Warning" posted!

**Comment by yoinx** [comment]
@ninadpchaudhari sounds right. I mentioned vaguely that if this integration was ever to go for an official status, that this specific attribute would need to be removed anyway as it shouldn't be there. I just didn't elaborate on it in the discussion I guess.
